lambdas/select-cri/src/main/resources/log4j2.yaml

```diff
@@ -0,0 +1,19 @@
+Configuration:
```

Comment:
I think you might be missing the `IpvLamdbaJsonLayout.json` file that goes with this. You can find it in other lambdas Resource dir

Reply:
Done!